### PR TITLE
Lots of code_80071F00 matches

### PR DESCRIPTION
**Pull request by Tmcg2**
Also removed a bunch of fake variables related to the object list.

### CODE REVIEW

File: src/code_8006E9C0.h [hunk 1]
```diff
@@ -23,6 +35,10 @@ extern s8  D_8018EDF3;
 extern u8 *D_8018D1E0;
 extern intptr_t D_8018D9B0;
 
+// Destination for decompressed exhaust textures (I think)
+// I'm not how much I like this "pointer to an array" setup, but if it matches it matches
+extern u8 (*D_8018D220)[1024];
```
Comment (MegaMech) [inline src/code_8006E9C0.h:40]: I could see this being like

`extern ExhaustTexture D_8018D220[num_exhause_textures];`

**Review by MegaMech — APPROVED** [review]

**Comment by MegaMech** [comment]
Please resolve conflicts when you get a chance to do so.

**Comment by Tmcg2** [comment]
Rebased, conflicts resolved

File: data/data_code_80071F00.s [hunk 2]
```diff
@@ -874,13 +874,16 @@ glabel D_800E6724
 glabel D_800E672C
 .word D_800E659C, D_800E6668
 
+# These should be floats, really
+# I believe they're the positions of the static neon signs in rainbow road
```
Comment (MegaMech) [inline data/data_code_80071F00.s:878]: Can you add todo:

I know we don't really use todo's. But this helps it be searchable later.

**Review by MegaMech — APPROVED** [review]

**Pull request merged**